Gopkg.toml

```diff
@@ -69,8 +69,6 @@ required = [
   name = "github.com/knative/serving"
   version = "v0.2.1"
 
-# STANZAS BELOW ARE GENERATED AND MAY BE WRITTEN - DO NOT MODIFY BELOW THIS LINE.
```

Comment:
???

Reply:
We do not auto-generate anymore. This line comes from kubebuilder